doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -328,6 +328,10 @@ Cellular samples
 
    * Fixed an issue with devicetree configuration after HCI updates in `sdk-zephyr`_.
 
+* :ref:`pdn_sample` sample:
+
+  * Added dynamic PDN info.
```

Comment:
```suggestion
  * Added dynamic PDN information.
```
or 
* Added information about dynamic PDN.

Please check the meaning.

Reply:
updated